lib/screens/chatscreen/chat_database/database.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+
+class ChatDatabase {
+  final _auth = FirebaseAuth.instance;
+
+  final CollectionReference chatLists = Firestore.instance.collection('chatroom');
+  final CollectionReference group = Firestore.instance.collection('group');
+
+  Future<void> createChatRoom(String docId, String uid, String destination) async {
+    await chatLists.document(docId).setData({
+      'lastMessage': Timestamp.now(),
+      'destination': destination,
+      'users': FieldValue.arrayUnion([uid]),
```

Comment:
Please add this also in the pad. Don't edit anything but add it at the bottom. Title name - "New database schema objects"